skyuxconfig.json

```diff
@@ -5,6 +5,9 @@
   "skyuxModules": [
     "SkyErrorModule"
   ],
+  "moduleAliases": {
+    "@skyux/core": "./src/app/public"
```

Comment:
This is causing my a circular dependency on my local `skyux serve`. It looks like the culprit is `i18nModule` which is importing `SkyAppWindowRef` from `@skyux/core`.

Reply:
Fix for the `@skyux/i18n` library incoming...